app/models/service_ansible_playbook.rb

```diff
@@ -3,7 +3,11 @@ class ServiceAnsiblePlaybook < ServiceGeneric
 
   # A chance for taking options from automate script to override options from a service dialog
   def preprocess(action, add_options = {})
-    _log.info("Override with new options: #{add_options}") unless add_options.blank?
+    unless add_options.blank?
+      _log.info("Override with new options:")
+      $log.log_hashes(add_options)
```

Comment:
I agree a header with `log_hashes` is not helpful. So let's stick with the mixture of both types of logger for now.